### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: pytype/directors_test.py [hunk 1]
```diff
@@ -305,8 +305,8 @@ def check_warning(message_regex, text):
       self.assertEqual(1, error.lineno)
       self.assertRegex(str(error), message_regex)
 
-    check_warning("Unknown pytype directive.*disalbe.*",
-                  "# pytype: disalbe=test-error")
+    check_warning("Unknown pytype directive.*disable.*",
```
Comment (rchen152) [inline pytype/directors_test.py:308]: I'm pretty sure this is an intentional misspelling - the test is making sure that pytype reports an error when "disable" is spelled wrong.

**Review by rchen152 — CHANGES_REQUESTED** [review]
Thanks! Left one comment - I think the changes to directors_test.py need to be reverted.

File: pytype/directors_test.py (hunk 1 above)
Comment (kianmeng) [reply pytype/directors_test.py]: @rchen152 Noted. Changes reverted. Please check again.

**Review by rchen152 — APPROVED** [review]
Thanks again! I'm going to merge this PR as-is and fix the lint errors when importing it into the Google codebase. (That means they'll be resolved when I re-export.)

**Comment by kianmeng** [comment]
> Thanks again! I'm going to merge this PR as-is and fix the lint errors when importing it into the Google codebase. (That means they'll be resolved when I re-export.)

@rchen152 :+1: :tada:

**Pull request merged**